### DISCUSSION TITLE
List of AWS permissions needed to deploy Hubs Cloud

### DISCUSSION THREAD

**Discussion by mohrtw** [mozilla/hubs#3036] [Hubs Cloud Questions] [answered]
Could we get a list of AWS permissions needed to deploy Hubs Cloud? That way we could create a role specifically for deploying Hubs Cloud instead of just granting admin permissions.

**Comment by robinkwilson** [comment, marked as answer]
This is an amazon / AWS question since it pertains to what's the necessary permissions to buy services / deploy software from the AWS Marketplace. 

I'm not sure if AWS Marketplace offerings require granular permissions into the CloudFormation stack such as deploying EC2 instances, setup EFS, S3, set CNAMES on domains, all the Hubs Cloud stack dependencies etc. 

You'll need to check amazon.
